src/main/java/org/jsoup/select/Selector.java

```diff
@@ -117,20 +117,19 @@ public static Elements select(String query, Iterable<Element> roots) {
         Validate.notEmpty(query);
         Validate.notNull(roots);
         Evaluator evaluator = QueryParser.parse(query);
-        ArrayList<Element> elements = new ArrayList<>();
+        Elements elements = new Elements();
```

Comment:
`Elements` is already an `ArrayList`. The only reason to copy construct below is so it effectively `trims()` the list.